openedx/features/course_experience/views/course_outline.py

```diff
@@ -37,39 +37,23 @@ def render_to_fragment(self, request, course_id=None, page_context=None, **kwarg
         if not course_block_tree:
             return None
 
-        # TODO: EDUCATOR-2283 Remove 'show_visual_progress' from context
-        # and remove the check for it in the HTML file
-        show_visual_progress = (
-            completion_waffle.visual_progress_enabled(course_key) and
-            self.user_enrolled_after_completion_collection(request.user, course_key)
-        )
+        show_visual_progress = self.user_enrolled_after_completion_collection(request.user, course_key)
```

Comment:
Oh, yup, this is right. I wrote the comment to get rid of the entire variable before I added in user_enrolled_after_completion_collection, and never changed the comment to reflect that. #allcommentsarelies